ioc_module.js

```diff
@@ -1,12 +1,11 @@
-'use strict'
-
 const EmptyActivityEndpoint = require('./dist/commonjs/index').Endpoints.EmptyActivity;
 const EventEndpoint = require('./dist/commonjs/index').Endpoints.Event;
 const ExternalTaskEndpoint = require('./dist/commonjs/index').Endpoints.ExternalTask;
 const ManualTaskEndpoint = require('./dist/commonjs/index').Endpoints.ManualTask;
 const NotificationEndpoint = require('./dist/commonjs/index').Endpoints.Notification;
 const ProcessModelEndpoint = require('./dist/commonjs/index').Endpoints.ProcessModel;
 const UserTaskEndpoint = require('./dist/commonjs/index').Endpoints.UserTask;
+const TaskEndpoint = require('./dist/commonjs/index').Endpoints.Task;
```

Comment:
Das passt nicht mit dem Export in der `endpoints/index` zusammen.

```suggestion
const FlowNodeInstanceEndpoint = require('./dist/commonjs/index').Endpoints.FlowNodeInstance;
```